frontend/react-native/src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { TouchableOpacity, Text } from 'react-native';
+
+const Button = () => {
+  return (
+    <TouchableOpacity>
```

Comment:
Falta prop `onPress`, una forma de agregar todas las props sin tener que especificar una por una puede ser destructurar props dentro TouchableOpacity ejemplo
```suggestion

Button = props => {
  return (
    <TouchableOpacity {...props}>
```